cypress/e2e/Get_Stock_Items_By_Supplier_Name.cy.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
+describe('get_stock_items_by_supplier_name', () => {
+  it('passes', () => {
+    cy.visit('http://localhost:8081/stock')
+    cy.wait(3000)
+    cy.get('#supplierName').type('Paul{enter}')
```

Comment:
Perhaps add another step to count number of elements in result table for a more thorough testing?

Reply:
Sure, I can add it quickly.